examples/specs/normalized/trellis_row_column_normalized.vl.json

```diff
@@ -23,6 +23,14 @@
       "y": {
         "field": "Miles_per_Gallon",
         "type": "quantitative"
+      },
+      "color": {
+        "field": "Cylinders",
+        "type": "nominal"
+      },
+      "shape": {
```

Comment:
I see what you mean now.  I guess we should add some suffix to this file to suggest that it's a test example, not demo example.  (maybe `_test`) so we can preserve the original name for the docs.